RussianFilter/sections/specific.txt

```diff
@@ -49,6 +49,10 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+psychologies.ru##.webinar-banner
+psychologies.ru##img[src="https://www.psychologies.ru/i/blocks/psy_zen_promo_300x300.jpg"]
+psychologies.ru##.invite-banner
+psychologies.ru##.mt-3
```

Comment:
`psychologies.ru##.webinar-banner` not an ad.
`psychologies.ru##img[src="https://www.psychologies.ru/i/blocks/psy_zen_promo_300x300.jpg"]` exchange it to the url blocking rule and move it to the annoyances -> newsletters.
`psychologies.ru##.invite-banner` also not an ad.
`psychologies.ru##.mt-3` too wide rule.

Reply:
I made the necessary changes.